### ISSUE TITLE
render imgflip images

### ISSUE THREAD

**Issue by alltheseas** [damus-io/damus#2573]
![image](https://github.com/user-attachments/assets/d8e6c0c0-ec64-4697-9215-d8fc8fdd24d5)

https://damus.io/nevent1qqsruzqcggn2kpyqnhvh0crhglz2mvz6z864rqykxzqwhg8x7s4ljjqpp4mhxue69uhkummn9ekx7mqpz4mhxue69uhk2er9dchxummnw3ezumrpdejqz9rhwden5te0wfjkccte9ejxzmt4wvhxjmcpzfmhxue69uhhqatjwpkx2urpvuhx2ucz4yyxs

**Comment by SanjaySiddharth** [comment]
Hello @alltheseas , please take a look at the screenshot  below (it didn't accept the screen recording) :

![Image](https://github.com/user-attachments/assets/157b9ddb-0895-4080-a1d8-6f4f94140fc0)

Let me know if this works , will create a PR .

**Comment by alltheseas** [comment]
Image looks good. Just remove the URL

**Comment by alltheseas** [comment]
@SanjaySiddharth consider also webm ticket: https://github.com/damus-io/damus/issues/2090

**Comment by SanjaySiddharth** [comment]
Hello @alltheseas , let me try removing the url

**Comment by SanjaySiddharth** [comment]
yes, i noticed this anomaly when I tested it . Thanks, I will reach out to you if I need some help

**Comment by SanjaySiddharth** [comment]
Hello @alltheseas , I think it works fine now . Please check and let me know

https://github.com/user-attachments/assets/e397770d-30a0-487f-a9fc-26aac9b63436

**Comment by SanjaySiddharth** [comment]
Hello @alltheseas ,please review

**Comment by SanjaySiddharth** [comment]
Hello @alltheseas , can we close this issue as this has been resolved in https://github.com/damus-io/damus/pull/2890 ?

**Comment by alltheseas** [comment]
Closing as part of a backlog triage — we're narrowing focus to the most actionable issues to reduce noise for contributors.

If this is still relevant, please reopen with updated context or reproduction steps. Thanks for reporting!
